rotkehlchen/db/dbhandler.py

```diff
@@ -3061,8 +3061,8 @@ def add_rpc_node(self, node: WeightedNode) -> None:
                 )
             except sqlcipher.IntegrityError as e:  # pylint: disable=no-member
                 raise InputError(
-                    f'Node for {node.node_info.blockchain} with name {node.node_info.name} '
```

Comment:
Why not just do insert or ignore now?

Reply:
because we identify rpc_nodes by a integer primary key and I didn't know a replace removed the old one and kept the new one when is a check what triggers the conflict